_layouts/default.html

```diff
@@ -7,12 +7,32 @@
     {% include body-js.html %}
     {% include header.html %}
 
-    <div class="page-content">
+    <div id="page-content" class="page-content">
       <div class="wrapper">
         {{ content }}
       </div>
       {% include footer.html %}
     </div>
 
+    <script>
+        <!-- Execute after the DOM is loaded -->
+        (function() {
+          var content = document.getElementById('page-content');
+          content.addEventListener('scroll', function(e) {
+              var distanceY = window.pageYOffset || content.scrollTop,
+                  shrinkOn = 300,
+                  header = document.querySelector("header.post-header");
```

Comment:
By shrinking `header.post-header` itself, `.back-to-home` will also shrink because its width and height is defined in terms of relative `em` units. Perhap change `.back-to-home` width and height to `rem` units? Or do we want the back button to become smaller also?